cosmos/dbt/graph.py

```diff
@@ -195,6 +195,9 @@ def run_dbt_ls(
         if self.project.dbt_vars:
             ls_command.extend(["--vars", yaml.dump(self.project.dbt_vars)])
 
+        if self.render_config.selector:
```

Comment:
We should raise an exception - somewhere - if a user tries to use this argument with a load method other than `dbt ls`.

Reply:
Added this to `DbtGraph` for manifest and custom load methods along with a test to check that an exception is raised.